src/Sylius/Bundle/CoreBundle/Fixture/BookProductFixture.php

```diff
@@ -93,6 +93,10 @@ public function load(array $options): void
                     'name' => 'Books',
                 ],
             ],
+            'images' => [
+                [sprintf('%s/../Resources/fixtures/%s', __DIR__, 'books.jpg'), 'main'],
```

Comment:
This how it would look like in YAML:

```yml
images:
  - ["path.jpg", "thumbnail"]
```

I guess instead of indexed array it's better to use associative array:

```yml
images:
  - { path: "foo.jpg", type: "thumbnail" }
```